frontend/src/components/loginOutButton/index.tsx

```diff
@@ -1,17 +1,12 @@
 import React from 'react';
+import { Button } from '@material-ui/core';
```

Comment:
I thought we went for `import { Button } from '@material-ui/core/Button';` to keep the bundle size small, since we're not using babel?

Reply:
Woops! React did this automatically and I thought I'd changed all of these to import in that way.